airflow/utils/log/s3_task_handler.py

```diff
@@ -45,7 +45,7 @@ def _build_hook(self):
         except Exception:
             self.log.error(
                 'Could not create an S3Hook with connection id "%s". '
-                'Please make sure that airflow[s3] is installed and '
```

Comment:
I think the `_api` suffix is unncessary and only existed for GCP as "api" is in the name of the google package.

Reply:
I think that it is worth introducing some naming consequences, but I will not insist on it. In the case of Googel, it should not exist either, because it installs many libraries and only one contains word `api` in that case.

Reply:
In the case of GCP, it also does not make sense. Maybe it's worth deleting in every case? If we make changes, let's do them to the end. There will be one note in the update procedure.

Reply:
Everything is an API - so the suffix is redundant to my mind.

Reply:
I apply your suggestion.

Reply:
I was thinking we would change `gcp_api` to just `gcp`. What do you think of that?

Reply:
The name change for GCP will have many opponents. I would like to do it in a separate PR to collect opinions. With this PR, I would like to reduce the number of packages. In another PR I reported that the Azure developers are very creative and I want to introduce a certain order so that they can introduce their changes. When this change is not accepted, they are blocked because they have open comments without a correct solution. See: https://github.com/apache/airflow/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+Azure

Reply:
I can not create a new PR now, because the new PR has to change the note from `UPDATING.MD'